src/main/java/com/firebase/geofire/DatasnapshotGeoQueryEventListener.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Firebase GeoFire Java Library
+ *
+ * Copyright © 2017 Firebase - All Rights Reserved
```

Comment:
Let's leave the licenses untouched for now, since Firebase was bought by Google I am not cure if `Copyright 2017 Firebase` actually means anything ... if anything we'd want to change the Copyright to Google 2017 but we can leave that out of this PR and I will try and address it across the codebase once I find out the answer.